privaterelay/utils.py

```diff
@@ -57,6 +63,23 @@ def get_countries_info_from_lang_and_mapping(
     }
 
 
+def get_subplat_upgrade_link_by_language(
+    accept_language: str, period: PeriodStr = "yearly"
+) -> str:
+    country_str = guess_country_from_accept_lang(accept_language)
+    country = cast(CountryStr, country_str)
+    language_str = accept_language.split("-")[0].lower()
+    language = cast(LanguageStr, language_str)
```

Comment:
If we keep this code, or use similar, we should create some functions that validate that the country string is one of the accepted countries. But, all of this works and we've handled cases where the country isn't in the mapping.